CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+| community            | The author of the pull request is not one of the project maintainers                    |
+| do-not-merge         | The pull request should not be merged by a project maintainer                           |
+| pr-changes-requested | A project maintainer has requested changes and is awaiting updates from the author      |
+| pr-expired           | Requested changes were not provided within an expected time window.                     |
+| pr-in-review         | A project maintainer is reviewing the pull request                                      |
+
+## Creating documentation and samples
+
+There are a variety of ways you can contribute to Functions content. In addition to those discussed in this section, updates to repository readmes, wikis, or code comments are always welcome.
+
+The primary Functions documentation on [docs.microsoft.com](https://docs.microsoft.com) is open source on [MicrosoftDocs/azure-docs](https://github.com/MicrosoftDocs/azure-docs). You can learn more about contributing to the docs at the repo or by following the [Microsoft Docs Contributor Guide](https://docs.microsoft.com/contribute/). 
+
+The [Serverless Community Library](https://serverlesslibrary.net) is a catalog of community samples that can be deployed right away. You can submit a sample to the library by following the [Serverless Library contribution guide](https://github.com/Azure/ServerlessLibrary#how-to-contribute-to-the-serverless-library). In addition to samples, most client experiences for creating functions make use of a function template. The [templates repository](Azure/azure-functions-templates) defines templates for each supported trigger and language, as well as core sample scenarios.
+
+## Azure Functions repositories
+
+Azure Functions is built on work from the following repositories:
+
+| Category            | Repository                                                                                                          | Description                                                                                                                    |
+|---------------------|---------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| General             | [Azure/azure-functions](https://github.com/Azure/azure-functions)                                                   | General information and issue management                                                                                       |
+| General             | [Azure/azure-functions-premium-plan](https://github.com/Azure/azure-functions-premium-plan)                         | Documentation and issue management related to the Azure Functions Premium Plan                                                 |
```

Comment:
I never ended up needing this repo. I should delete it 😬

Reply:
README will be updated as a secondary activity if that's ok. I want to get this one in place first.

Also, while we're on the topic of premium cleanup, can we get the stuff out of this repo?

Reply:
Yes but where should we move it to?